### ISSUE TITLE
error for not enough gpu 

### ISSUE THREAD

**Issue by benyamin-moraditabar** [sokrypton/ColabFold#532]
## Steps to Reproduce (for bugs)
Please make sure to reproduce the issue after a "Factory Reset" in Colab. 
If running locally ypdate you local installation `colabfold_batch` to the newest version.
Please provide your input if you can share it. 

## ColabFold Output (for bugs)

Not Enough GPU memory? INTERNAL: CustomCall failed: jaxlib/gpu/prng_kernels.cc:33: operation gpuGetLastError() failed: cudaGetErrorString symbol not found.
## Context

i have faced with this error
Not Enough GPU memory? INTERNAL: CustomCall failed: jaxlib/gpu/prng_kernels.cc:33: operation gpuGetLastError() failed: cudaGetErrorString symbol not found.

## Your Environment
Include as many relevant details about the environment you experienced the bug in.
* Git commit used
* If you run it on a local system. Please add the server specifications
* Operating system and version:

**Comment by milot-mirdita** [comment]
I pushed a fix, the batch notebook was still using jax 0.3, which broke with today's CUDA 12 upgrade of Google Colab.

**Comment by neyrinck** [comment]
Hi, I have been getting this error too on Colab, and still am, with apparently the default libraries on colab:
 jax==0.4.23
 jaxlib==0.4.23+cuda11.cudnn805
 colabfold==1.5.5 or 1.5.2

Is there some combination of libraries that does work with CUDA 12?

**Comment by milot-mirdita** [comment]
Which one of the notebook is giving this error?

**Comment by neyrinck** [comment]
Hi, sorry it wasn't one of the official colabfold notebooks, it was my modification of a version from several months ago of a notebook used by ChimeraX, https://github.com/RBVI/ChimeraX/blob/develop/src/bundles/alphafold/src/alphafold21_predict_colab.ipynb
My modified code stopped working around when this issue was originally posted, but it does appear that the current version of the ChimeraX notebook works, so it looks like I just have to make my modifications to that new version.

**Comment by neyrinck** [comment]
Hello, just a note that I was able to fix the error by making the same changes that ChimeraX did in https://github.com/RBVI/ChimeraX/blob/develop/src/bundles/alphafold/src/alphafold21_predict_colab.ipynb
In case this is useful to anyone else, these changes were to replace

`  from os.path import exists
  if not exists('COLABFOLD_READY'):
    print ('Installing ColabFold')
    cmds = f'''
set -e
# We have to use "--no-warn-conflicts" because colab already has a lot preinstalled with requirements different to ours
pip install --no-warn-conflicts "colabfold[alphafold-minus-jax] @ git+https://github.com/sokrypton/ColabFold@dc9fc3d03379d23784e796f4c7fd31d173bafaa2"
# high risk high gain
pip uninstall jaxlib -y
pip install "jax[cuda11_cudnn805]==0.3.24" jaxlib==0.3.24+cuda11.cudnn805 -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
touch COLABFOLD_READY
'''
    run_shell_commands(cmds, 'install_colabfold.sh', install_log)
`

with
`  from os.path import exists
  if not exists('COLABFOLD_READY'):
    print ('Installing ColabFold')
    cmds = f'''
set -e
# We have to use "--no-warn-conflicts" because colab already has a lot preinstalled with requirements different to ours
pip install --no-warn-conflicts "colabfold[alphafold-minus-jax] @ git+https://github.com/sokrypton/ColabFold@29395e7dee2c2895eaf585ef6b6cde932fd95e72"
# We use the Google Colab system jaxlib since it needs to be compatible with CUDA.
# Haiku works with jax and we need an updated version to work with jax newer than 0.3.25.
pip install --upgrade dm-haiku
# patch for jax > 0.3.25
sed -i 's/weights = jax.nn.softmax(logits)/logits=jnp.clip(logits,-1e8,1e8);weights=jax.nn.softmax(logits)/g' /usr/local/lib/python{python_version}/dist-packages/alphafold/model/modules.py
# pip uninstall jax jaxlib -y
# pip install "jax[cuda]==0.3.25" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
# Biopython 1.82 removes SCOPData required by AlphaFold.
pip install biopython==1.81
touch COLABFOLD_READY
'''
    run_shell_commands(cmds, 'install_colabfold.sh', install_log)`
